src/main/kotlin/com/autonomousapps/tasks/BuildHealthTask.kt

```diff
@@ -61,14 +61,18 @@ abstract class BuildHealthTask : DefaultTask() {
       if (consoleReport.isNotEmpty()) shouldPrintPath = true
 
       val consoleText = advicePrinter.consoleText()
-      buildHealthText
-        .appendReproducibleNewLine(projectHeaderText(projectAdvice.projectPath))
-        .appendReproducibleNewLine(consoleText)
-
-      // Only print to console if we're not configured to fail
-      if (!shouldFail) {
-        logger.quiet(projectHeaderText(projectAdvice.projectPath))
-        logger.quiet(consoleText)
+      if (consoleText.isNotEmpty()) {
```

Comment:
Note to self: run locally, check output.

@vlsi you might want to fix up `ProjectHealthTask` as well.